cms/djangoapps/contentstore/views/item.py

```diff
@@ -556,6 +571,9 @@ def _save_xblock(user, xblock, data=None, children_strings=None, metadata=None,
             data = old_content['data'] if 'data' in old_content else None
 
         if fields:
+            is_discussable = fields.pop('is_discussable', None)
+            set_discussion_toggle(is_discussable, xblock, user)
+
```

Comment:
@ahmedbilal  Can you please look at @Agrendalath 's PR for the correct location for this? it will be available in "metadata" not fields when operated by the studio frontend code.